zerver/openapi/zulip.yaml

```diff
@@ -1798,6 +1798,53 @@ paths:
                                   "upload_space_used": 0,
                                   "id": 0,
                                 }
+                            - type: object
+                              additionalProperties: false
+                              description: |
+                                Event sent to a user's clients when registration of their
+                                push device to bouncer service either succeeds or fails.
+
+                                Helps clients to live-update the `push_devices` dictionary
+                                returned in [`POST /register`](/api/register-queue) response.
+
+                                **Changes**: New in Zulip 11.0 (feature level ZF-1653f2).
+                              properties:
+                                id:
+                                  $ref: "#/components/schemas/EventIdSchema"
+                                type:
+                                  allOf:
+                                    - $ref: "#/components/schemas/EventTypeSchema"
+                                    - enum:
+                                        - push_device
+                                op:
+                                  type: string
+                                  enum:
+                                    - update
+                                push_account_id:
+                                  type: string
+                                  description: |
+                                    The push account ID whose registration to bouncer succeed or fail.
+                                status:
+                                  type: string
+                                  description: |
+                                    The updated registration status.
+                                    Either `"active"` or `"failed"`.
```

Comment:
It's probably useful to declare this as an `enum` here? Not sure.

Reply:
> It's probably useful to declare the valid values in the OpenAPI spec as an enum here rather than a string

Yeah, done.

> I think technically "pending" should be on this list.

The description is "Event sent to a user's clients when registration of their push device to bouncer service either succeeds or fails.", so including "pending" at this stage can be confusing. We should add when such a case occurs ?